openedx/features/enterprise_support/api.py

```diff
@@ -247,10 +248,11 @@ def fetch_enterprise_learner_data(self, site, user):
             querystring = {'username': user.username}
             response = endpoint().get(**querystring)
         except (HttpClientError, HttpServerError):
-            message = ("An error occurred while getting EnterpriseLearner data for user {username}".format(
-                username=user.username
-            ))
-            LOGGER.exception(message)
+            LOGGER.exception(
+                'Failed to get enterprise-learner for user [%s] with client user [%s]',
+                user.username,
+                self.user.username
```

Comment:
Is the thinking that these two values will be different at some point?

Reply:
Yes, they can be different. Sometimes we use the Enterprise service user, other times we use the request user. This will help us better understand who is most often getting throttled.